Source/Fluxor/DependencyInjection/LifecycleEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Fluxor
+{
+	public enum LifecycleEnum
```

Comment:
Please rename to `FluxorServiceLifetime`
I would have suggested just `ServiceLifeTime` but I think that might cause problems for users using both `Fluxor` and `Microsoft.Extensions.DependencyInjection` namespaces.

Reply:
done